tests/setup/jest_extend.ts

```diff
@@ -145,4 +150,83 @@ CancelledReasons: ${this.utils.printReceived(dispatchResult.reasons)}
       message,
     };
   },
+  toHaveValue(target: DOMTarget, expectedValue: string | boolean) {
+    const element = getTarget(target);
+    if (!(element instanceof HTMLInputElement) && !(element instanceof HTMLSelectElement)) {
+      const message = element ? "Target is not an input element" : "Target not found";
+      return { pass: false, message: () => message };
+    }
+    const value =
+      element instanceof HTMLInputElement && element.type === "checkbox"
+        ? element.checked
+        : element.value;
+    if (value !== expectedValue) {
+      return {
+        pass: false,
+        message: () =>
+          `expect(target).toHaveValue(expected);\n\n${this.utils.printDiffOrStringify(
+            expectedValue,
+            value,
+            "Expected value",
+            "Received value",
+            false
+          )}`,
+      };
+    }
+    return { pass: true, message: () => "" };
+  },
+  toHaveText(target: DOMTarget, expectedText: string) {
+    const element = getTarget(target);
+    if (!(element instanceof HTMLElement)) {
+      const message = element ? "Target is not an HTML element" : "Target not found";
+      return { pass: false, message: () => message };
+    }
+    const text = element.textContent;
+    if (text !== expectedText) {
+      return {
+        pass: false,
+        message: () =>
+          `expect(target).toHaveText(expected);\n\n${this.utils.printDiffOrStringify(
+            expectedText,
+            text,
+            "Expected text",
+            "Received text",
+            false
+          )}`,
+      };
+    }
+    return { pass: true, message: () => "" };
+  },
+  toHaveCount(selector: string, expectedCount: number) {
+    const elements = document.querySelectorAll(selector);
+    if (elements.length !== expectedCount) {
+      return {
+        pass: false,
+        message: () =>
+          `expect("${selector}").toHaveCount(expected);\n\n${this.utils.printDiffOrStringify(
+            expectedCount,
+            elements.length,
+            "Expected",
+            "Received",
+            false
+          )}`,
+      };
+    }
+    return { pass: true, message: () => "" };
+  },
 });
+
+function getTarget(target: DOMTarget): Element | Document | Window {
```

Comment:
I cannot import the helper from `test_helpers/dom`, it makes half of the tests crash if I import it ? Not sure why ....